cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubernetes-lifecycle-metrics
+  namespace: kube-system
+  labels:
+    application: kubernetes-lifecycle-metrics
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9090"
+    prometheus.io/scrape: "true"
```

Comment:
I guess these are not needed on the deployment?

Reply:
Ah, makes sense.